src/Symfony/Bridge/Twig/Resources/views/Form/bootstrap_4_layout.html.twig

```diff
@@ -234,3 +235,11 @@
     </div>
     {%- endif %}
 {%- endblock form_errors %}
+
+{# Help #}
+
+{% block form_help -%}
+    {% if help is not empty %}
+        <small class="form-text text-muted">{{ help|trans }}</small>
```

Comment:
Let's make sure it respects the translation domain. 

`{{ translation_domain is same as(false) ? help : help|trans({}, translation_domain) }}`

Reply:
It checks if `translation_domain` is false, it it is we just print `help`. If it is not false we use translation with that domain.

Reply:
I was thinking about TranslationDomain, I don't know what it is, I'd watch.

Reply:
Have a look here: http://symfony.com/doc/current/components/translation.html#using-message-domains

It is basically a way to categorize translation messages.

Make sure you add this fix every time you print `help`

Reply:
Done :-)